src/lib.rs

```diff
@@ -66,9 +66,6 @@
 #![cfg_attr(feature="clippy", deny(clippy))]
 #![cfg_attr(feature="clippy", allow(use_debug))]
 
-#[cfg(test)]
```

Comment:
I thought the policy was to do all crate imports here for all modules, even test-only ones?